features/AssetController.feature

```diff
@@ -17,6 +17,33 @@ Feature: Device Manager asset controller
       | _id      | "outils-PERFO-asset_01" |
     Then The document "engine-kuzzle":"assets":"outils-PERFO-asset_01" does not exists
 
+  Scenario: Delete a linked asset
+    When I successfully execute the action "device-manager/asset":"create" with args:
+      | engineId       | "engine-ayse" |
+      | body.type      | "outils"        |
+      | body.model     | "PERFO"         |
+      | body.reference | "asset_02"      |
+    Then The document "engine-ayse":"assets":"outils-PERFO-asset_02" exists
```

Comment:
This step is not useful, the previous step will throw an error if anything goes wrong because if the `successfully` keyword

Reply:
done